core/components/SearchBar/SearchBar.tsx

```diff
@@ -1,13 +1,17 @@
 import { Strings } from "@/constants/strings";
 import AppTitle from "@/shared/typography/app_title";
+import React from "react";
 import { useSearchBar } from "./hooks/useSearchBar";
 import { FormWrapper, InputFieldWrapper, InputWrapper } from "./styled";
 
 const SearchBar = () => {
   const { countSearchedTasks, onChange } = useSearchBar();
 
   return (
-    <FormWrapper onSubmit={() => event?.preventDefault()}>
+    <FormWrapper
+      onSubmit={(event: React.SyntheticEvent) => event?.preventDefault()}
```

Comment:
no need to check if event have value

```suggestion
      onSubmit={(event: React.SyntheticEvent) => event.preventDefault()}
```